test/test_utils/Mocks.h

```diff
@@ -21,7 +22,7 @@ using namespace yarpl::flowable;
 /// UnmanagedMockPublisher's lifetime MUST be managed externally.
 template <typename T>
 class MockFlowable : public Flowable<T> {
- public:
+public:
```

Comment:
I have used the "format code" function of Nuclide editor and got this output.
Let me run clang-format.